lib/process-services/src/lib/process-list/components/process-list/process-list.component.ts

```diff
@@ -85,6 +85,9 @@ export const processPresetsDefaultModel = {
     templateUrl: './process-list.component.html'
 })
 export class ProcessInstanceListComponent extends DataTableSchema implements OnChanges, AfterContentInit, PaginatedComponent {
+    @ViewChild(DataTableComponent)
```

Comment:
Why do we need this change?

Reply:
Reverted those changes, as I've found a possibly better solution